### PR TITLE
View my merchant dashboard page showing my order fulfillment

### PR DESCRIPTION
**Pull request by codesrobertson**
merchant#show
order_item#index

Order fulfillment link

Nested Route

### CODE REVIEW

File: app/controllers/merchants_controller.rb [hunk 1]
```diff
@@ -1,4 +1,16 @@
 class MerchantsController < ApplicationController
+
+  def show
```
Comment (cojenco) [inline app/controllers/merchants_controller.rb:5]: Once we merge branches, we can apply the controller filter #require_login onto the merchant actions 👍

File: app/models/merchant.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class Merchant < ApplicationRecord 
   has_many :order_items
   has_many :products
+  has_many :orders
+
```
Comment (cojenco) [inline app/models/merchant.rb:5]: Not sure if we connected this relation. The team can discuss on this Monday. If so, I think we will need to run a migration.

**Review by cojenco — APPROVED** [review]
Thanks for creating the merchant dashboard and order fulfillment page. LGTM👍

File: app/models/merchant.rb (hunk 2 above)
Comment (aecombs) [reply app/models/merchant.rb:5]: I think this needs to be one of those distinct associations -- merchant has many orders through order_items, something along those lines.

**Review by aecombs — APPROVED** [review]
GO FOR IIIIITTT
But also write tests eventually
BUT GO FOR IT

**Pull request merged**